### ISSUE TITLE
Tooltip not working as expected with markdown and matrix

### ISSUE THREAD

**Issue by ft-lie** [surveyjs/survey-library#3850]
### Are you requesting a feature, reporting a bug or asking a question?

bug/unexpected feature

### What is the current behavior?

The artefact can be seen by e.g. mouse over the dog. the html tags appears as text in the tooltip/title when hovering.

https://surveyjs.io/Examples/Library?id=survey-markdown-radiogroup&platform=jQuery&theme=bootstrap

### What is the expected behavior?

it would be nice to specify separately what should be presented as tooltip and what should be presented as (html) option. 

e.g. for an option specify 

```
{
    name : 'id1',
    text : 'as it is today, used where other options does not make sense',
    markDown : 'feed into markdown where html would be nice', 
    html : '<b>preformatted</b> html code as alternative to markDown',
    otherInfo : 'markdown should receive the entire object, so even this property could',
    otherinfo2: 'be used for rendering',
}

```
### How would you reproduce the current behavior (if this is a bug)?
(see link above)

### Specify your

* browser: chrom, opera
* browser version: 82
* surveyjs platform (angular or react or jquery or knockout or vue): jquery bootstrap 
* surveyjs version: 9.

**Comment by dszewczyk1** [comment]
@dmitrykurmanov @tsv2013  I don't see any response here and I can see it is a problem for me as well. I'm using markdown and in the "title" there is HTML visible, which looks awful.

![image](https://user-images.githubusercontent.com/9110884/150313908-68efab22-d348-40c9-9b43-e8e62809eb59.png)

Is there a way not to show this "title" at all for radio/checkbox as it just shows the same, what is alrerady visible for the option?

**Comment by dmitrykurmanov** [comment]
we will investigate the problem and write back asap

**Comment by tsv2013** [comment]
We decided to remove tooltip as redundant due to item text is always visible. Will be included in v1.9.9.

**Comment by dszewczyk1** [comment]
@dmitrykurmanov @tsv2013 I can see the 'title' is removed but something connected to it... There's `aria-label` in the `<label>` which includes all the HTML from markdown and this results in reading the iimg' HTML tag by screen reader... 
Can be seen here https://surveyjs.io/Examples/Library?id=survey-markdown-radiogroup&platform=Reactjs&theme=defaultV2
